src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,48 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+
 public class WorkWithFile {
+    private static final int SUM_OPERATION = 1;
 
     public void getStatistic(String fromFileName, String toFileName) {
+        File file = new File(fromFileName);
+        int supply = 0;
+        int buy = 0;
+
+        try (BufferedReader br = new BufferedReader(new FileReader(file))) {
+            String line;
+
+            while (true) { // loop for reading data from file while end
+                line = br.readLine();
+                if (line == null) { // check null
+                    break;
+                }
```

Comment:
```suggestion
            while ((line = br.readLine()) != null) { 
```
давай скоротиму тут код